modules/home/classes/transformations-manager.php

```diff
@@ -35,14 +35,14 @@ class Transformations_Manager {
 	public function __construct( $home_screen_data ) {
 		$container = Plugin::$instance->elementor_container();
 
-		if ( $container->has( Wordpress_Adapter_Interface::class ) ) {
-			$this->wordpress_adapter = $container->get( Wordpress_Adapter_Interface::class );
+		if ( $container->has( Wordpress_Adapter::class ) ) {
```

Comment:
Why we need to change this part from Interface to access directly to class? Is there any issue with Interface?

Reply:
In PHP-DI, interfaces are not resolved automatically unless you explicitly define them in the config so I removed them

Reply:
Is autowire does not help?

Reply:
No autowire does not resolve interfaces by default. Interfaces can be implemented to multiple concrete classes so DI needs to be told when it's resolving a certain interface which class to resolve

Reply:
try to var_dump `$container->has( Wordpress_Adapter_Interface::class )` to see that it always returns false until you create a config to bind the interface to class